packages/twenty-server/src/engine/metadata-modules/field-metadata/utils/validate-default-value-for-type.util.ts

```diff
@@ -69,13 +69,24 @@ export const validateDefaultValueForType = (
       FieldMetadataClassValidation
     >(validator, conputedDefaultValue as FieldMetadataClassValidation);
 
-    return (
-      validateSync(defaultValueInstance, {
-        whitelist: true,
-        forbidNonWhitelisted: true,
-        forbidUnknownValues: true,
-      }).length === 0
-    );
+    const errors = validateSync(defaultValueInstance, {
```

Comment:
@magrinj Is it ok or shall we use a logger ?

Reply:
I think we're ok here, but what you've done is only logging in the console.
If you want to be more specific on the error sent by the backend to the front-end, you can host the errors in the `IsFieldMetadataDefaultValue`, I'll give you an example below.